Jenkinsfile

```diff
@@ -294,6 +294,18 @@ pipeline {
                 org.camunda.helper.CIAnalytics.trackBuildStatus(this, userReason)
             }
         }
+        failure {
+            script {
+                if (env.BRANCH_NAME != 'develop' || agentDisconnected()) {
+                    return
+                }
+
+                echo "Send slack message"
+                slackSend(
```

Comment:
In case of a change to failure this will now send two notifications. One here and one below.

Reply:
Makes sense 👍